packages/fuel-gauge/src/coverage-contract.test.ts

```diff
@@ -556,6 +561,22 @@ describe('Coverage Contract', () => {
     expect(value).toStrictEqual(OUTPUT);
   });
 
+  it('should test mixed native enum [Native->NotNative]', async () => {
+    const input = MixedNativeEnum.Native;
+    const expected = { NotNative: MixedNativeEnum.NotNative };
+
+    const { value } = await contractInstance.functions.mixed_native_enum(input).call();
+    expect(value).toStrictEqual(expected);
+  });
+
+  it('should test mixed native enum [NotNative->Native]', async () => {
+    const input = { NotNative: MixedNativeEnum.NotNative };
+    const expected = 'Native';
+
+    const { value } = await contractInstance.functions.mixed_native_enum(input).call();
+    expect(value).toStrictEqual(expected);
+  });
+
```

Comment:
Those aren't necessary because the tests are already covering both enum variants as inputs and outputs. Having `Native->Native` and `NonNative->NonNative` would be redundant in what the tests are trying to achieve: testing out the encoding and decoding logic.

Reply:
Agreed with @nedsalk